lib/cnet/eth/eth_tx.c

```diff
@@ -16,30 +16,33 @@
 
 #include <cnet_node_names.h>
 #include "eth_tx_priv.h"        // for eth_tx_node_ctx_t, ETH_TX_NEXT_MAX
+#include "chnl_callback_priv.h"
 
 static struct eth_tx_node_main eth_tx_main;
 
 static uint16_t
-eth_tx_node_process(struct cne_graph *graph, struct cne_node *node, void **objs, uint16_t nb_objs)
+eth_tx_node_process(struct cne_graph *graph __cne_unused, struct cne_node *node, void **objs,
+                    uint16_t nb_objs)
 {
     eth_tx_node_ctx_t *ctx = (eth_tx_node_ctx_t *)node->ctx;
     uint16_t port          = ctx->port; /* Get TX port id */
     uint16_t count         = nb_objs;
 
-    CNE_SET_USED(graph);
```

Comment:
Did you have a reason to remove this line and add __cne_unused? Maybe some compiler issue.